fastify.js

```diff
@@ -591,10 +592,15 @@ function wrapRouting (httpHandler, { rewriteUrl, logger }) {
 }
 
 function loadVersion () {
+  versionLoaded = true
   const fs = require('fs')
   const path = require('path')
-  const pkg = JSON.parse(fs.readFileSync(path.join(__dirname, 'package.json')))
-  return pkg.version
+  const pkgPath = path.join(__dirname, 'package.json')
+  if (fs.existsSync(pkgPath)) {
```

Comment:
https://nodejs.org/api/fs.html#fs_fs_accesssync_path_mode

Reply:
Using `fs.accessSync` requires wrapping the block in a try/catch since that function throws an error on failure rather than returning a `boolean` like `fs.existsSync`. 

Even addressing that, this change does not resolve the issue.  I just tried with this change and the following error is outputted:

```
fastify not found, proceeding anyway
FastifyError [Error]: fastify-plugin: fastify-compress - expected '3.x' fastify version, '%s' is installed
    at Object.checkVersion (webpack-internal:///./node_modules/fastify/lib/pluginUtils.js:96:63)
    at Object.registerPlugin (webpack-internal:///./node_modules/fastify/lib/pluginUtils.js:110:16)
    at Boot.override (webpack-internal:///./node_modules/fastify/lib/pluginOverride.js:28:57)
    at Plugin.exec (webpack-internal:///./node_modules/avvio/plugin.js:79:33)
    at Boot.loadPlugin (webpack-internal:///./node_modules/avvio/plugin.js:266:10)
    at Task.release (webpack-internal:///./node_modules/fastq/queue.js:140:16)
    at worked (webpack-internal:///./node_modules/fastq/queue.js:182:10)
    at eval (webpack-internal:///./node_modules/avvio/plugin.js:269:7)
    at done (webpack-internal:///./node_modules/avvio/plugin.js:201:5)
    at check (webpack-internal:///./node_modules/avvio/plugin.js:225:9)
    at internal/process/task_queues.js:153:7
    at AsyncResource.runInAsyncScope (async_hooks.js:186:9)
    at AsyncResource.runMicrotask (internal/process/task_queues.js:150:8)
    at processTicksAndRejections (internal/process/task_queues.js:97:5) {
  code: 'FST_ERR_PLUGIN_VERSION_MISMATCH',
  statusCode: 500
}
```